packages/core/helper-plugin/src/components/AnErrorOccurred.tsx

```diff
@@ -2,10 +2,24 @@ import React from 'react';
 
 import { EmptyStateLayout } from '@strapi/design-system';
 import { ExclamationMarkCircle } from '@strapi/icons';
-import PropTypes from 'prop-types';
-import { useIntl } from 'react-intl';
+import { PrimitiveType, useIntl } from 'react-intl';
 
-const AnErrorOccurred = ({ content, ...rest }) => {
+export type AnErrorOccurredProps = {
+  content?: {
+    id: string;
+    defaultMessage: string;
+    values?: Record<string, PrimitiveType>;
+  };
```

Comment:
Would this be better as an `MessageDescriptor & { values?: Record<string, PrimitiveType> }`?